profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module

```diff
@@ -0,0 +1,327 @@
+    );
+    $element['fs_objects']['fs_prepopulate']['description_populate'] = array(
+      // '#type' => 'select_or_other',
+      '#type' => 'select',
+      '#title' => t('Popup description'),
+      '#options' => $options,
+      '#description' => t('Choose a field to populate the popup description'),
+      // '#other' => t('Free value'),
+      '#default_value' => isset($value['fs_objects']['fs_prepopulate']['description_populate']) ? $value['fs_objects']['fs_prepopulate']['description_populate'] : 'body',
+    );
+  }
+  return $element;
+}
+
+/**
+ * Check if a form value is decimal with 5 decimals maximum.
+ */
+function _decimal_check($element, &$form_state) {
+  if (!empty($element['#value'])) {
+    if (!(preg_match('/^[0-9]+(\\.[0-9]{1,5})?$/', $element['#value']))) {
+      form_error($element, t('The field "!name" is not a decimal (with 5 decimals maximum).', array('!name' => $element['#title'])));
+    }
+  }
+}
+
+
+/**
+ * Implements hook_form_ID_alter().
+ */
+function nexteuropa_geofield_form_field_ui_field_edit_form_alter(&$form, &$form_state, $form_id) {
+
+  if ($form_state['build_info']['args'][0]['widget']['type'] == 'geofield_widget_map') {
+    $form['field']['cardinality']['#default_value'] = 1;
+    $form['field']['cardinality']['#disabled'] = TRUE;
+    $form['instance']['default_value_widget']['#collapsible'] = TRUE;
+    $form['instance']['default_value_widget']['#collapsed'] = TRUE;
+    $form['instance']['default_value_widget']['#access'] = FALSE;
+  }
+}
+
+/**
+ * Implements hook_field_formatter_info().
+ */
+function nexteuropa_geofield_field_formatter_info() {
+
+  return array(
+    'geofield_geojson_map_format' => array(
+      'label' => t('Interactive map'),
+      'field types' => array('geofield_geojson'),
+    ),
+    'geofield_geojson_raw' => array(
+      'label' => t('geoJson data'),
+      'field types' => array('geofield_geojson'),
+    ),
+  );
+}
+
+/**
+ * Implements hook_field_formatter_view().
+ */
+function nexteuropa_geofield_field_formatter_view($entity_type, $entity, $field, $instance, $langcode, $items, $display) {
+
+  $element = array();
+  $settings = $instance['widget']['settings'];
+
+  switch ($display['type']) {
+    case 'geofield_geojson_raw':
+        $element[0] = array(
+          '#type' => 'html_tag',
+          '#tag' => 'div',
+          '#attributes' => array(
+            'id' => 'geofield_geojson_raw',
+          ),
+          '#value' => isset($items[0]['geofield_geojson']) ? $items[0]['geofield_geojson'] : '',
+        );
+      break;
+
+    case 'geofield_geojson_map_format':
+        $value = isset($items[0]['geofield_geojson']) ? $items[0]['geofield_geojson'] : '';
+      $element[0] = array(
+        '#type' => 'markup',
+        '#markup' => '<div id="geofield_geojson_map"></div><div id="geofield_geojson_map_wrapper"></div>',
+      );
+      $element[0]['#attached']['js'][0] = array(
```

Comment:
Page not found : 
[...]profiles/multisite_drupal_standard/libraries/leaflet/leaflet-src.js?nugaoq

So the map is not available on the content form...